recipe/meta.yaml

```diff
@@ -1,13 +1,15 @@
 {% set name = "pdfstream" %}
-{% set version = "0.5.2" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "dac32f481ced3b002e02b5efde508bf02afe5a3bb278261d18f6ea0f02e762fa" %}
```

Comment:
There are a few changes to the grayskull-provided template. Can you please run the command from https://github.com/conda-forge/staged-recipes#grayskull---recipe-generator-for-python-packages-on-pypi to make it more compatible? Thanks!

Reply:
I will check it out. Thank you.